src/components/calcite-input-message/readme.md

```diff
@@ -27,13 +27,12 @@ Displays a contextual message to a user. Allows the passing of content, links, e
 | `icon`   | `icon`    | when used as a boolean set to true, show a default icon based on status. You can also pass a calcite-ui-icon name to this prop to display a custom icon | `boolean \| string`              | `undefined` |
 | `scale`  | `scale`   | specify the scale of the input, defaults to m                                                                                                           | `"l" \| "m" \| "s"`              | `"m"`       |
 | `status` | `status`  | specify the status of the input field, determines message and icons                                                                                     | `"idle" \| "invalid" \| "valid"` | `"idle"`    |
-| `type`   | `type`    | specify the appearance of any slotted message - default (displayed under input), or floating (positioned absolutely under input)                        | `"default" \| "floating"`        | `"default"` |
 
 ## CSS Custom Properties
 
 | Name                                    | Description                                                                                                                          |
 | --------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------ |
-| `--calcite-input-message-spacing-value` | when type is default, the top margin above the input-message. When type is floating, the amount of padding around the input-message. |
+| `--calcite-input-message-spacing-value` | The top margin above the input-message.                                                                                              |
```

Comment:
You can leave out this file change (the readmes are updated automatically via GitHub action running the build script)